### DISCUSSION TITLE
Artisan command `make:auth` not found

### DISCUSSION THREAD

**Discussion by virtual-designer** [laravel/framework#40005] [Q&A] [answered]
Hi, I have installed laravel with composer:

```
composer create-project laravel/laravel myapp
```

But when I run `php artisan make:auth` is throws an error (It fails to recognize the command).

How to fix this issue?

**Comment by dinhquochan** [comment, marked as answer]
Laravel remove command `make:auth`. You can install package `laravel/ui` it's included `make:auth`. But, It is outdate. You should use`laravel/breeze` package or `laravel/jetstream` package instead.
